tests/api-watch/test_diff_modtpe_decl.ml

```diff
@@ -0,0 +1,68 @@
+open Api_watch
+open Test_helpers
+
+let%expect_test "Module_type with value changes" =
+  let reference =
+    compile_interface
+      {|
+    module type M = sig
+      val g : int -> string
+    end
+  |}
+  in
+  let current = compile_interface {|
+    module type M = sig end
+  |} in
+  let result = Diff.interface ~module_name:"Main" ~reference ~current in
+  Format.printf "%a" pp_diff_option result;
+  [%expect
+    {| Some (Module Main: {Modified (Supported [ Module M: {Modified (Supported [ Value (g, Removed)])}])}) |}]
```

Comment:
The first two tests are testing the same thing, i.e. a module type whose signature has been modified. You can delete this one.